modules/ROOT/pages/update-acm.adoc

```diff
@@ -53,3 +53,18 @@ Attempting to upgrade the package without this feature enabled causes the proces
 This starts the installation process.
 
 When the upgrade finishes, you receive a notification email.
+
+If you are upgrading from package version 1.62 or before, after installing, ensure both `Member` and `Guest` profiles have field-level `Read` access to the `ParentId` field in the `CommunityAPI` object:
+
+. `Member` profile:
```

Comment:
Member and Guest probably be bulleted lists items since they don't mention specific tasks? If you want to use number lists, I would suggest making them into steps.

Reply:
Good point! I made these into steps.